seeds/ledgerAccountType copy.js

```diff
@@ -0,0 +1,75 @@
+/*****
```

Comment:
Please fix this filename or remove - I'm not sure if this was accidentally copied.